src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -129,38 +140,16 @@ const DatePickerMenu: FC<Props> = ({onCollapse, timeRange, timeRangeLabel}) => {
       }
     }
     setDateRange([start, end])
-    let startInput = start
-    let endInput = end
-
-    const pad = part => part.toString().padStart(2, '0')
-    if (startInput instanceof Date) {
-      const dateParts = [
-        startInput.getFullYear(),
-        startInput.getMonth() + 1,
-        startInput.getDate(),
-      ]
-        .map(pad)
-        .join('-')
-      const timeParts = [startInput.getHours(), startInput.getMinutes()]
-        .map(pad)
-        .join(':')
-      startInput = `${dateParts} ${timeParts}`
+
+    // format for input display in DatePicker
+    let startInput: string = null
+    let endInput: string = null
+
+    if (start instanceof Date) {
+      startInput = moment(inputStart).format(FORMAT)
     }
-    if (endInput instanceof Date) {
-      const endDate = new Date(endInput)
-      // this sets the time to the end of the selected end day
-      endDate.setMinutes(endDate.getMinutes() - 1)
-      const dateParts = [
-        endDate.getFullYear(),
-        endDate.getMonth() + 1,
-        endDate.getDate() + 1,
-      ]
-        .map(pad)
-        .join('-')
-      const timeParts = [endDate.getHours(), endDate.getMinutes()]
-        .map(pad)
-        .join(':')
-      endInput = `${dateParts} ${timeParts}`
```

Comment:
All this to avoid using moment lib. Such that our own code will need to be debugged for all the same problems. 🤦🏼‍♀️